pkgs/development/python-modules/numpy/default.nix

```diff
@@ -91,6 +89,9 @@ in buildPythonPackage (rec {
   nativeBuildInputs = [ gfortran cython ];
   buildInputs = [ blas lapack ];
 
+  # Causes `error: argument unused during compilation: '-fno-strict-overflow'` due to `-Werror`.
+  hardeningDisable = lib.optionals stdenv.cc.isClang [ "strictoverflow" ];
```

Comment:
We can't set the warn-error flag from here https://github.com/numpy/numpy/blob/0aaa5d397f86d5b4aef9c83053f65296a790e501/numpy/distutils/command/build.py#L17 somehow?

Reply:
That's more of a discussion for the original PR.  Here I just move the line to simplify the expression.

I would expect that some `-Wno-error` approach would also work.

Reply:
Doesn’t it make more sense to stop passing the unused argument than to hack at the build scripts to suppress the unused argument warning?

See #243595 regarding `-fno-strict-overflow` and Clang.

Reply:
Ah, sorry.